updateutils/update.py

```diff
@@ -0,0 +1,166 @@
+
+    @commands.group(name="jojoutils")
+    async def jojo_utils(self, ctx: commands.Context):
+        """Base command for checking/updating Jojo's utils"""
+        pass
+
+    @jojo_utils.command(name="check")
+    async def jojo_utils_check_updates(self, ctx: commands.Context):
+        """Check if you need to update Jojo's utils"""
+
+        version = await self._get_version()
+        description = "You are up to date!"
+        if jojoutils_version != version:
+            description = (
+                f"You are out of date! "
+                f"You are on version {jojoutils_version} but version {version} is available!"
+            )
+        kwargs = {"content": f"Jojo Utils updater\n\n{description}"}
+        if await ctx.embed_requested():
+            embed = discord.Embed(
+                title="Jojo Utils updater",
+                description=description,
+                colour=await ctx.embed_colour(),
+            )
+            embed.set_thumbnail(
+                url="https://raw.githubusercontent.com/Just-Jojo/JojoCog-Assets/main/JOJO_COGS.png"
+            )
+            embed.timestamp = datetime.utcnow()
+            kwargs = {"embed": embed}
+        await ctx.send(**kwargs)
+
+    @jojo_utils.command(name="version")
+    async def jojo_utils_version(self, ctx: commands.Context):
+        """Shows the version of the cog and Jojo's utils"""
+
+        latest_version = await self._get_version()
+        data = {
+            "Cog Version": self.__version__,
+            "Jojo Utils version": jojoutils_version,
+            "Latest version on GitHub": latest_version,
+        }
+        kwargs = {
+            "content": f"Jojo Utils updater\n\n"
+            + "\n".join(f"**{k}** {v}" for k, v in data.items())
+        }
+        if await ctx.embed_requested():
+            embed = discord.Embed(
+                title="Jojo Utils updater",
+                description="Versions!",
+                colour=await ctx.embed_colour(),
+            )
+            embed.set_thumbnail(
+                url="https://raw.githubusercontent.com/Just-Jojo/JojoCog-Assets/main/JOJO_COGS.png"
+            )
+            [embed.add_field(name=k, value=v, inline=False) for k, v in data.items()]
+            embed.timestamp = datetime.utcnow()
+            kwargs = {"embed": embed}
+        await ctx.send(**kwargs)
+
+    @jojo_utils.command(name="update")
+    async def jojo_utils_update(self, ctx: commands.Context):
+        """Attempt to update your version of Jojo's utils"""
+
+        latest_version = await self._get_version()
+        if jojoutils_version == latest_version:
+            await ctx.send("Your version of Jojo's utils is up to date")
+            return
+        pred = MessagePredicate.yes_or_no(ctx)
+        msg = await ctx.send(
+            "This is the command that will be ran. Are you okay with this (input y/n)?"
+            f"\n\n`<path to your venv's python> {self.command}`"
```

Comment:
```suggestion
            f"\n\n`{self.path} {self.command}`"
```
I feel like this should contain the path